pkgs/desktops/deepin/deepin-terminal/default.nix

```diff
@@ -52,12 +54,20 @@ stdenv.mkDerivation rec {
     patchShebangs .
   '';
 
-  enableParallelBuilding = true;
+  cmakeFlags = [
+    "-DTEST_BUILD=OFF"
+    "-DUSE_VENDOR_LIB=OFF"
+    "-DVERSION=${version}"
+  ];
+
+  postInstall = ''
+    ln -s deepin-terminal "$out"/bin/x-terminal-emulator
```

Comment:
I think that's a debian packaging thing that we don't need to have symlinked here.
(though please tell if otherwise)

Reply:
Probably it is. But if I am not wrong, some forthcoming deepin packages refers to `x-terminal-emulator`. But that can be fixed too.

Reply:
Removed the symlink.

Reply:
Yeah I do see some references
* https://github.com/linuxdeepin/dde-file-manager/blob/4d861c9bafa67015d30487162753de5b669b2e13/dde-file-manager-lib/shutil/fileutils.cpp#L658
* https://github.com/linuxdeepin/dde-file-manager/blob/4d861c9bafa67015d30487162753de5b669b2e13/dde-file-manager-lib/shutil/fileutils.cpp#L658

Probably something for later though.